src/neo/SmartContract/ApplicationEngine.Blockchain.cs

```diff
@@ -10,12 +10,12 @@ partial class ApplicationEngine
     {
         public const uint MaxTraceableBlocks = Transaction.MaxValidUntilBlockIncrement;
 
-        public static readonly InteropDescriptor System_Blockchain_GetHeight = Register("System.Blockchain.GetHeight", nameof(GetBlockchainHeight), 0_00000400, TriggerType.Application, CallFlags.AllowStates, true);
-        public static readonly InteropDescriptor System_Blockchain_GetBlock = Register("System.Blockchain.GetBlock", nameof(GetBlock), 0_02500000, TriggerType.Application, CallFlags.AllowStates, true);
-        public static readonly InteropDescriptor System_Blockchain_GetTransaction = Register("System.Blockchain.GetTransaction", nameof(GetTransaction), 0_01000000, TriggerType.Application, CallFlags.AllowStates, true);
-        public static readonly InteropDescriptor System_Blockchain_GetTransactionHeight = Register("System.Blockchain.GetTransactionHeight", nameof(GetTransactionHeight), 0_01000000, TriggerType.Application, CallFlags.AllowStates, true);
-        public static readonly InteropDescriptor System_Blockchain_GetTransactionFromBlock = Register("System.Blockchain.GetTransactionFromBlock", nameof(GetTransactionFromBlock), 0_01000000, TriggerType.Application, CallFlags.AllowStates, true);
-        public static readonly InteropDescriptor System_Blockchain_GetContract = Register("System.Blockchain.GetContract", nameof(GetContract), 0_01000000, TriggerType.Application, CallFlags.AllowStates, true);
+        public static readonly InteropDescriptor System_Blockchain_GetHeight = Register("System.Blockchain.GetHeight", nameof(GetBlockchainHeight), 0_00000400, CallFlags.AllowStates, true);
```

Comment:
If someone calls this method under Verification, it could cause mismatch between nodes, so some nodes can verify as true, others as false, then we can't use this to ban this node, in future we should qualify and ban the nodes if they provide incorrect or unverifiable information.

Reply:
> If someone calls this method under Verification, it could cause mismatch between nodes, so some nodes can verify as true, others as false

This is not true. If a node is not synchronized to the latest block, the `Verification` it performs is meaningless. Because when it synchronizes the new block, it will need to be verified again.

Reply:
Yes, you are right, it could happend without this syscall too